packages/node/src/integrations/tracing/dataloader.ts

```diff
@@ -0,0 +1,51 @@
+import { DataloaderInstrumentation } from '@opentelemetry/instrumentation-dataloader';
+import {
+  SEMANTIC_ATTRIBUTE_SENTRY_OP,
+  SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN,
+  defineIntegration,
+  spanToJSON,
+} from '@sentry/core';
+import type { IntegrationFn } from '@sentry/types';
+import { generateInstrumentOnce } from '../../otel/instrument';
+
+const INTEGRATION_NAME = 'Dataloader';
+
+export const instrumentDataloader = generateInstrumentOnce(INTEGRATION_NAME, () => new DataloaderInstrumentation({}));
```

Comment:
I think we should set `requireParentSpan` to be `true` by default, and then expose config to change it accordingly, but I'm open to other opinions.

@mydea what do you think?

Reply:
Let's set `requireParentSpan` to be `true` by default @onurtemizkan. We don't need to expose config for now, we can add that later based on user request.

Reply:
Just realised it's also `true` by default in `DataloaderIntegration`. Still, passed `true` from our side, and we can expose it later if we need to.

Reply:
this sounds reasonable to me, I guess you rarely want this to be a transaction!